### ISSUE TITLE
Docker build creates a broken container

### ISSUE THREAD

**Issue by benhylau** [benhylau/ssbc-grants-scuttlemesh#16]
The current Dockerfile builds a container that fails to start:

```
root@dundas:~/work# docker logs ssb
[28-Jul-2018 05:58:03] WARNING: Nothing matches the include pattern '/etc/php5/fpm.d/*.conf' from /etc/php5/php-fpm.conf at line 15.
Starting SBOT
/root/ssb-web-pi/start-docker.sh: line 22: sbot: not found
```

Have you tested a full run since https://github.com/benhylau/ssbc-grants-scuttlemesh/pull/13/files or did you make changes to ssb-web-pi repo in ways that would break this?

**Comment by benhylau** [comment]
Fix https://github.com/benhylau/ssbc-grants-scuttlemesh/pull/17

**Comment by benhylau** [comment]
The broken docker image was due to none of the sbot stuff being built. The build completed very quickly (< 5 min) without any of the sbot compiling, which generally takes about half an hour. It may have been due to temporary DNS failures since I was experiencing failures on the Raspberry Pi host, and now that I have changed to 8.8.8.8, `master` is building again.

I am curious though why the docker build continued as if there weren't errors, and only at runtime sbot is missing.
